domain/src/main/java/life/qbic/domain/usermanagement/EmailFormatException.java

```diff
@@ -0,0 +1,14 @@
+package life.qbic.domain.usermanagement;
+
+import life.qbic.apps.datamanager.ApplicationException;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
```

Comment:
What causes this exception to be thrown? What is the meaning of this exception?

Reply:
I removed it, it seems to be a remnant from before the emailvalidation exception days